src/server/contest/runner.py

```diff
@@ -12,7 +12,7 @@ class Runner():
     """To compile, execute and evaluate the
     submitted code against saved testcases """
     BASE_TEST_CASES_DIR = cwd / 'bank/testcases'
-    BASE_SUBMISSION_DIR = cwd / 'contest/submissions'
+    BASE_SUBMISSION_DIR = cwd / 'contest/submissions/latest'
```

Comment:
shouldn't this have been a new variable? `BASE_SUBMISSION_DIR` shouldn't be the path to latest submissions.